### ISSUE TITLE
Poorly combined characters in hOCR pane; font issue?

### ISSUE THREAD

**Issue by n8marti** [manisandro/gImageReader#613]
If I use a keyboard that combines a base character with an accent (e.g. U+0069 U+0300, or i + ̀ ), the hOCR document pane does a poor job combining the decomposed characters into one. In the following image the first "ì" is generated with the above sequence from one keyboard, while the second one is generated from a different keyboard, which outputs the single composed character U+00EC:
![image](https://user-images.githubusercontent.com/23388501/214516903-298b78ce-8f46-47ad-ba9d-edf54f540bd4.png)

I believe this must be an issue with the display font used in the hOCR window. Is there a way I can change that font to test out this theory? This is problematic because if I export the hOCR document as PDF using a font that is known to correctly render the characters in other apps (e.g. LibreOffice), they're still incorrectly rendered in the exported PDF.

I was getting similar poor results in gnome-terminal and gedit before changing the fonts used in both places to DejaVu Sans Mono from the default.

**Comment by n8marti** [comment]
Well, I think I confirmed this in a more general way. I changed the Desktop font in dconf editor from "Ubuntu 11" to a compatible font "Charis SIL Compact 11". This indeed fixes the problem. Below is the same text, but now retyped with the changed Desktop font:
![image](https://user-images.githubusercontent.com/23388501/214779381-2fcb714d-c8b3-4d6f-90d6-4c29d05dc8af.png)

Is there any way gImageReader could add an option to change the app display font? Maybe I need to contact Ubuntu and see if they can improve their font rendering.
